packages/turf-area/index.ts

```diff
@@ -23,10 +23,13 @@ export default function area(
   return geomReduce(
     geojson,
     (value, geom) => {
+      if (geom == null) {
```

Comment:
I think it mentioned being able to represent features that don't currently have a location/geographic representation, but will.

Reply:
I think null is sent to the iterators so the caller can decide what to do with it.

If it's helpful, I made some observations on broader null handling (or lack thereof) in #2154, which I closed because it took on too much.

Reply:
You'll also see in that PR's comments that the core Geometry types (from @types/geojson) conveniently don't require that we handle null.  So I left Turf's mixed handling, realizing it could take some substantial effort to streamline.

https://github.com/Turfjs/turf/pull/2154#discussion_r678543842